portfolio/src/main/webapp/script.js

```diff
@@ -70,19 +70,16 @@ function addRandomFact() {
  * Fills in the 'I am a' sentences with different phrases
  */
 function writeSnippets() {
-    if (i == 0) document.getElementById('snippetsTarget').innerHTML = EMOJIS[j] + ' I am ';
+    if (charIdx == 0) document.getElementById('snippetsTarget').innerHTML = EMOJIS[phraseIdx] + ' I am ';
```

Comment:
```suggestion
    if (charIdx == 0) {
        document.getElementById('snippetsTarget').innerHTML = EMOJIS[phraseIdx] + ' I am ';
    }
```
Also - don't use innerHTML
(I used 4 spaces, but use a tab or whatever you've already been doing - tab in this editor doesn't do insert a tab)